types.go

```diff
@@ -95,10 +95,23 @@ type Histogram interface {
 	Start() Stopwatch
 }
 
-// Stopwatch is a helper for simpler tracking of elapsed time.
-type Stopwatch interface {
-	// Stop records the difference between the current clock and start time.
-	Stop()
+// Stopwatch is a helper for simpler tracking of elapsed time, use the
+// Stop() method to report time elapsed since its created back to the
+// timer or histogram.
+type Stopwatch struct {
+	Start    time.Time
```

Comment:
I was originally going to do that but I thought this may be more useful as the type can be used outside the package for return from a subpackage type.